stdlib/3/smtplib.pyi

```diff
@@ -39,71 +44,77 @@ class SMTPConnectError(SMTPResponseException): ...
 class SMTPHeloError(SMTPResponseException): ...
 class SMTPAuthenticationError(SMTPResponseException): ...
 
-def quoteaddr(addrstring): ...
-def quotedata(data): ...
+def quoteaddr(addrstring: str) -> str: ...
+def quotedata(data: str) -> str: ...
 
 class SMTP:
     debuglevel = ...  # type: int
+    # Type of file should match what socket.makefile() returns
     file = ...  # type: Any
-    helo_resp = ...  # type: Any
-    ehlo_msg = ...  # type: Any
-    ehlo_resp = ...  # type: Any
-    does_esmtp = ...  # type: Any
-    default_port = ...  # type: Any
+    helo_resp = ...  # type: Optional[bytes]
+    ehlo_msg = ...  # type: str
+    ehlo_resp = ...  # type: Optional[bytes]
+    does_esmtp = ...  # type: bool
```

Comment:
Actually an int (although used like a bool).

Reply:
I actually have a PR open on that -- it's also documented to be used as a bool.
https://github.com/python/cpython/pull/8718

Reply:
...but will change to int in the meantime.